apps/ledger-live-mobile/src/families/celo/WithdrawFlow/WithdrawAmount.tsx

```diff
@@ -0,0 +1,378 @@
+                  const disabled = withdrawalTime > new Date();
+                  return (
+                    transaction.index === index ? (
+                      <Selectable selected={true} name={formatAmount(value)} hasClock={disabled} />
+                    ) : (
+                        <Touchable onPress={disabled ? () => openModal(time) : () => onChange(index)}>
+                            <Selectable selected={false} name={formatAmount(value)} hasClock={disabled} />
+                          </Touchable>
+                    )
+                  )
+              })) : <Text> Something went wrong, can't fetch any funds to withdraw </Text>}
+          </View>
+        </View>
+        
+          <View style={styles.footer}>
+            <View style={styles.feesRow}>
+                <SendRowsFee
+                  account={account}
+                  parentAccount={parentAccount}
+                  transaction={transaction}
+                />
+              </View>
+            <Button
+              event="CeloWithdrawAmountContinue"
+              type="primary"
+              title={
+                <Trans
+                  i18nKey={
+                    !bridgePending
+                      ? "common.continue"
+                      : "send.amount.loadingNetwork"
+                  }
+                />
+              }
+              containerStyle={styles.continueButton}
+              onPress={onContinue}
+              disabled={bridgePending || !!bridgeError || hasErrors}
+              pending={bridgePending}
+            />
+          </View>
+          <BottomModal isOpened={!!infoModalOpen} onClose={closeModal}>
+        <View style={styles.modal}>
+          <View style={styles.infoIcon}>
+            <InfoIcon bg={colors.lightLive}>
+              <Clock color={colors.live} size={30} />
+            </InfoIcon>
+          </View>
+          <View style={styles.infoRow}>
+            <LText style={[styles.warnText, styles.title]} semiBold>
+              <Trans
+                i18nKey={`celo.withdraw.flow.steps.amount.waitingTitle`}
+              />
+            </LText>
+            <LText style={styles.warnText} color="grey">
+              <Trans
+                i18nKey={`celo.withdraw.flow.steps.amount.waitingPeriod`}
+                values={{ date: getDate(infoModalOpen.toString()) }}
+              />
+            </LText>
+          </View>
+        </View>
+      </BottomModal>
+      </SafeAreaView>
+  );
+}
+
+const Line = ({ children }: { children: ReactNode }) => (
+  <View style={styles.summaryLine}>{children}</View>
+);
+
+const Words = ({
+  children,
+  highlighted,
+  style,
+}: {
+  children: ReactNode;
+  highlighted?: boolean;
+  style?: any;
+}) => (
+  <Text
+    numberOfLines={1}
+    fontWeight={highlighted ? "bold" : "semiBold"}
+    style={[styles.summaryWords, style]}
+    color={highlighted ? "live" : "smoke"}
+  >
+    {children}
+  </Text>
+);
+
+
+const Selectable = ({
```

Comment:
Can we try to mutualize this component with the one in celo/components directory ?